include/utils.h

```diff
@@ -145,6 +149,48 @@ inline int delete_file(const std::string& fileName) {
   }
 }
 
+inline void convert_labels_string_to_int(const std::string& inFileName,
+  const std::string& outFileName,
+  const std::string& mapFileName,
+  const std::string& unv_label) {
+  std::unordered_map<std::string, _u32> string_int_map;
+  std::ofstream label_writer(outFileName);
+  std::ifstream label_reader(inFileName);
+  if(unv_label != "")
+    string_int_map[unv_label] = 0;
+  std::string line, token;
+  while (std::getline(label_reader, line)) {
+    std::istringstream new_iss(line);
+    std::vector<_u32>lbls;
+    while (getline(new_iss, token, ',')) {
+        token.erase(std::remove(token.begin(), token.end(), '\n'), token.end());
+        token.erase(std::remove(token.begin(), token.end(), '\r'), token.end());
+        if (string_int_map.find(token) == string_int_map.end()) {
+            int nextId = string_int_map.size()+1;
```

Comment:
Question: when universal label is set, will any label have value 1? Because size of string_int_map will be 1, so label numbers will start from 2.

Reply:
yes it will start from 2. But its internal map anyways (does it have any side effect on building and searching ?)